### DISCUSSION TITLE
请问密码必须加密的原因

### DISCUSSION THREAD

**Discussion by Herissmon** [jeessy2/ddns-go#1229] [Q&A]
如题 本人利用1p构建本项目 部署已完成 但后来密码忘了 找配置文件已经找到密码 可惜加密 经查证无法解密 就想请问一下为啥非得加密……难道docker配置文件也不安全？🤔另外问下没有密码加密的版本号是多少 是否支持cloudflare

**Comment by Herissmon** [comment]
issues已经提交 但是被强制关闭 给个说法！

**Reply by jeessy2** [reply]
你想要什么说法😓

**Reply by Herissmon** [reply]
很简单 为啥被强制关闭issues😓

**Reply by jeessy2** [reply]
> 很简单 为啥被强制关闭issues😓

因为不能回退，不可能为了小部分人发牢骚的影响其他没发声的

**Comment by jeessy2** [comment]
加密作用：防止密码重用攻击，很大一部分人使用的一个主密码，如果主密码泄露了，其他系统就危险了，所以主流网站都会加密，并不可反解析。

使用什么版本自行查看，先搜索在提问。
